<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scorche</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="viewPort">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>System</string>
    </property>
    <addaction name="actionRendering"/>
    <addaction name="actionShow_console"/>
    <addaction name="actionDebug_AI"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew_game"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionBots_enable_quick_aim"/>
    <addaction name="actionExplosion_rocks"/>
    <addaction name="actionFast_game"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuOptions"/>
   <addaction name="menuSystem"/>
  </widget>
  <action name="actionRendering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rendering</string>
   </property>
  </action>
  <action name="actionBots_enable_quick_aim">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bots - enable quick aim</string>
   </property>
  </action>
  <action name="actionNew_game">
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="actionExplosion_rocks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Explosion rocks</string>
   </property>
  </action>
  <action name="actionFast_game">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fast game</string>
   </property>
  </action>
  <action name="actionShow_console">
   <property name="text">
    <string>Show console</string>
   </property>
  </action>
  <action name="actionDebug_AI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug AI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
